### PR TITLE
ESSNTL 4821 inventory groups application in rbac call

### PR DESCRIPTION
**Pull request by petrblaho**
- Adds inventory to list of application to RBAC call
- Cleans up some comments

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [x] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by petrblaho** [comment]
https://issues.redhat.com/browse/ESSNTL-5249

File: kerlescan/view_helpers.py [hunk 1]
```diff
@@ -171,11 +171,14 @@ def ensure_has_permission(**kwargs):
...
         perms = get_perms(
-            kwargs["application"],
+            ",".join(
+                [kwargs["application"], "inventory"]
+            ),  # we need to query permissions for the inventory app
             auth_key,
             logger,
             kwargs["request_metric"],
             kwargs["exception_metric"],
+            kwargs.get("flask_g", None),
```
Comment (vkrizan) [inline kerlescan/view_helpers.py:181]: Do we need to pass the whole Flask's global to this method? Can we just pass what's necessary?

Comment (petrblaho) [reply kerlescan/view_helpers.py]: Good catch, thank you. Solved.

**Review by vkrizan — APPROVED** [review]
LGTM

**Pull request merged**